projects/igniteui-angular/src/test.ts

```diff
@@ -15,20 +15,24 @@ import localeFr from '@angular/common/locales/fr';
 import localeDe from '@angular/common/locales/de';
 import localeJa from '@angular/common/locales/ja';
 
-declare const require: any;
+declare const require: {
+  context(path: string, deep?: boolean, filter?: RegExp): {
+    <T>(id: string): T;
+    keys(): string[];
+  };
+};
```

Comment:
Guessing this was added before the project even had `@types/webpack-env`, which is where this seems to be coming. So the manual typing can be dropped whenever I guess.

Reply:
@damyanpetev This is still scaffolded by Angular in the latest CLI version. Check your own project generation in the `igniteui-cli`, and you'll find it.